vdp/pipeline/v1beta/pipeline.proto

```diff
@@ -1938,3 +1939,193 @@ message LookUpPipelineAdminResponse {
+  string pipeline_run_id = 2 [(google.api.field_behavior) = REQUIRED];
+
+  // The page number to retrieve.
+  optional int32 page = 3 [(google.api.field_behavior) = OPTIONAL];
+
+  // The maximum number of items per page to return. The default and cap values
+  // are 10 and 100, respectively.
+  optional int32 page_size = 4 [(google.api.field_behavior) = OPTIONAL];
+}
+
+// ListComponentRunsResponse is the response message for ListComponentRuns.
+message ListComponentRunsResponse {
+  // The list of component runs.
+  repeated ComponentRun component_runs = 1 [(google.api.field_behavior) = OUTPUT_ONLY];
+
+  // The total number of component runs matching the request.
+  int64 total_size = 2 [(google.api.field_behavior) = OUTPUT_ONLY];
+
+  // The current page number.
+  int32 page = 3 [(google.api.field_behavior) = OUTPUT_ONLY];
+
+  // The number of items per page.
+  int32 page_size = 4 [(google.api.field_behavior) = OUTPUT_ONLY];
+}
+
+// FileReference represents metadata for a file.
+message FileReference {
+  // Name of the file.
+  string name = 1 [(google.api.field_behavior) = REQUIRED];
+
+  // Format of the file (e.g., PDF, TXT, JPG).
+  string type = 2 [(google.api.field_behavior) = REQUIRED];
+
+  // Size of the file in bytes.
+  int64 size = 3 [(google.api.field_behavior) = REQUIRED];
+
+  // URL of the file (e.g., S3 URL).
+  string url = 4 [(google.api.field_behavior) = REQUIRED];
+}
+
+// PipelineRun represents a single execution of a pipeline.
+message PipelineRun {
+  // Unique identifier for the pipeline.
+  string pipeline_uid = 1;
+
+  // Unique identifier for each run.
+  string pipeline_run_uid = 2;
+
+  // Pipeline version used in the run.
+  string pipeline_version = 3;
+
+  // Current status of the run.
+  RunStatus status = 4;
+
+  // RunSource defines the source of a pipeline trigger
+  enum RunSource {
+    // Unspecified
+    RUN_SOURCE_UNSPECIFIED = 0;
+    // model trigger from frontend UI
+    RUN_SOURCE_CONSOLE = 1;
+    // model trigger from API or SDK
+    RUN_SOURCE_API = 2;
+  }
+
+  // Origin of the run.
+  RunSource source = 5;
+
+  // Time taken to complete the run.
+  optional google.protobuf.Duration total_duration = 6 [
+    (google.api.field_behavior) = OUTPUT_ONLY,
+    (google.api.field_behavior) = OPTIONAL
+  ];
+
+  // Identity of the user who initiated the run.
+  string requester_id = 7 [(google.api.field_behavior) = OUTPUT_ONLY];
+
+  // Input files for the run.
+  repeated FileReference inputs = 9;
+
+  // Output files from the run.
+  repeated FileReference outputs = 10;
+
+  // Snapshot of the pipeline recipe used for this run.
+  google.protobuf.Struct recipe_snapshot = 11;
+
+  // Time when the run was triggered.
+  google.protobuf.Timestamp triggered_time = 12;
+
+  // Time when the run started execution.
+  google.protobuf.Timestamp started_time = 13;
+
+  // Time when the run completed.
+  google.protobuf.Timestamp completed_time = 14;
```

Comment:
Consistency with other similar fields in the API (I'd rather have `started_at` but consistency > personal preference)

```suggestion
  google.protobuf.Timestamp start_time = 13;

  // Time when the run completed.
  google.protobuf.Timestamp complete_time = 14;
```